dojo/tools/nmap/parser.py

```diff
@@ -96,6 +97,7 @@ def get_findings(self, file, test):
                             "**Extra Info:** {}\n".format(port_element.find("service").attrib["extrainfo"])
                         )
                     description += service_info
+                script_id = "NoID"
```

Comment:
Why is this preferrable to just setting `vuln_id_from_tool = None` (the default) if there is no ID?

Reply:
You are right